README.md

```diff
@@ -124,6 +124,57 @@ Using JSON payload for constructing a message is also available:
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+### Update the message
```

Comment:
This section should be under the "Usage" section heading, which is also h3. Thus, adding this section heading might be a bit confusing to readers. Instead, I would suggest having h4:

```suggestion
#### Update the message
```